lib/util/ids.js

```diff
@@ -0,0 +1,6 @@
+var i  = 0
+module.exports = function (returnVal) {
+  if (returnVal) return 'js-ids-' + i
```

Comment:
@watilde I use it for InputRadio. However, I'm reviewing `<InputRadio/>` and then the option can be removed.